### DISCUSSION TITLE
Who is using Flink CDC ？

### DISCUSSION THREAD

**Discussion by leonardBang** [apache/flink-cdc#1080] [General]
Who's using/谁在使用 Flink CDC?

The Flink CDC community appreciates every user and developer who uses and cares about Flink CDC. We are committed to listening to each one of you to make the community more prosperous.
Flink CDC 社区感激每一位使用和关注 Flink CDC 的用户和开发者，我们努力倾听每一个人的声音，让社区变得更加繁荣。

So, the starting point for this topic is to encourage you, as a user or developer, to leave your real-world scenarios so that we can subsequently explore new and better solutions, schedule the releases, and solve real-world problems based on these scenarios!
所以，这个议题的出发点是希望作为用户或者开发者的您留下您的实际使用场景，以便我们后续基于此探索更好的解决方案，制定版本计划，解决实际问题。

We look forward to your comments in the comments space, including but not limited to the following areas.
我们期待您能在评论区留言，其内容包括但不限于以下方面。

Organization / 组织: e.g. Aliyun Cloud.
Region / 地区: e.g. Hangzhou, China
Contact / 联系方式: e.g. [leonard@apache.org](mailto:yao@apache.org)
Usage Scenario / 使用场景: e.g. We use Flink CDC to capture database changes from MySQL/PostgreSQL/MongoDB to Hologres/Kafka/ADB in Alibaba Cloud Realtime Compute for Apache Flink(https://www.aliyun.com/product/bigdata/sc).

Best Regards / 致敬,

Flink CDC  Community

**Comment by GOODBOY008** [comment]
![logo-big](https://user-images.githubusercontent.com/13617900/163749561-eb830a0e-911a-4b18-b723-0a1ddbe94825.png)

- Organization / 组织: GigaCloud Technology.
- Region / 地区: Suzhou, China
- Contact / 联系方式: [gongzhongqiang@gigacloudtech.com](gongzhongqiang@gigacloudtech.com)
- Usage Scenario / 使用场景: Use Flink CDC as source to build real-time datalake for bussiness analysis. We also build a data-sync system to sync data between applications by Flink CDC.

**Comment by lykpass** [comment]
![viewfile](https://user-images.githubusercontent.com/48595535/163793482-8f214a20-b5d8-4046-8079-ee1b2db6202a.png)
Organization / 组织: Zhongshang Huimin.
Region / 地区: Beijing, China
Contact / 联系方式: liyakun@huimin100.cn
Usage Scenario / 使用场景: We use Flink CDC to synchronize MySQL business data, and use Flink SQL to widen the data structure to build a real-time data warehouse

**Comment by fuyun2024** [comment]
![顺丰科技Logo-黑底白字](https://user-images.githubusercontent.com/26080592/167366986-9e5ceb97-34cc-49b2-95ae-73156f337a2d.jpg)

Organization / 组织: SF-EXPRESS
Region / 地区: Shenzhen, China
Contact / 联系方式: [qinlihui@sfmail.sf-express.com](qinlihui@sfmail.sf-express.com)
Usage Scenario / 使用场景: We use Flink CDC as the basis of data integration platform to capture change data from MYSQL/ORACLE and write it to Hudi and Kafka in real time

**Comment by rmetzger** [comment]
![image](https://user-images.githubusercontent.com/89049/201025738-b7bc262a-4dcf-40f2-92dc-cd0533f46a8c.png)

Organization: [Decodable](https://www.decodable.co/)
Region: USA
Usage Scenario: We use the flink-cdc-connectors for some of our [CDC Connectors](https://www.decodable.co/connectors) in Decodable. Decodable Connectors allow customers to ingest data from various sources into our SaaS Platform without worrying about configuration or operation of Flink clusters. We are very satisfied with the stability of the flink-cdc-connectors.

**Comment by r00mz** [comment]
Organization : 旺店通
Region : China
Usage Scenario: We use Flink CDC to MySQL RDS 2 MySQL RDS(Many to Many).

**Comment by Jiabao-Sun** [comment]
Organization / 组织: XTransfer
Region / 地区: Shanghai, China
Contact / 联系方式: [jiabao.sun@xtransfer.cn](mailto:jiabao.sun@xtransfer.cn)
Usage Scenario / 使用场景: We use Flink CDC to build real-time Data Warehouse and OLAP platform.

**Comment by Juneryang** [comment]
Organization / 组织: 竞技世界
Region / 地区: BeiJing, China
Contact / 联系方式: yangjun01@mail.jj.cn
Usage Scenario / 使用场景: Mysql(logTable and statusTable)、postgreSql、mariadb、mongodb cdc connector。

**Comment by zhongqishang** [comment]
![image](https://user-images.githubusercontent.com/16421989/201258623-ede8fb09-6d56-4571-aa27-feb458e83383.png)

Organization / 组织: 企查查
Region / 地区: Suzhou, China
Contact / 联系方式: zhongqishang@gmail.com
Usage Scenario / 使用场景: We use Flink CDC to capture database changes from MySQL/MongoDB to Kafka/Iceberg/ElasticSearch/Starrocks.

**Comment by Penny-Fei** [comment]
Organization / 组织: 
![2020_盖雅横式logo-512-原色](https://user-images.githubusercontent.com/45064825/201292314-e2e162a1-a193-42bf-97c2-a91abe29a894.jpg)

Region / 地区: Suzhou, China
Contact / 联系方式: [630374168@qq.com]
Usage Scenario / 使用场景: We used flinkcdc to synchronize data from different database servers for MySQL/Sqlserver/MongoDB to Kafka/ElasticSearch/Starrocks.

**Comment by lollipopcs** [comment]
<img width="542" alt="20221220-174306@2x" src="https://user-images.githubusercontent.com/35026750/208635519-4134a3e1-ec37-4e31-9240-203c6f2087c6.png">

Organization / 组织: 同程数科
Region / 地区: Shanghai, China
Contact / 联系方式: [song2.chen@ly.com](mailto:song2.chen@ly.com)
Usage Scenario / 使用场景: We use Flink CDC as the basis of data integration platform to sync MariaDB business data to Doris/Kafka to build real-time Data Warehouse and OLAP platform.

**Comment by ysq5202121** [comment]
![image](https://github.com/ververica/flink-cdc-connectors/assets/8586973/cab5d79e-ef00-40b3-8c78-02eda540bff1)
Organization / 组织: 华杰科技
Region / 地区: Shenzhen, China
Contact / 联系方式: 329629067@qq.com
Usage Scenario / 使用场景: We use SQL Server CDC to synchronize to clickhouse for BI report display

**Comment by csuzj** [comment]
Organization / 组织: 中芯国际（绍兴）
Region / 地区: Shaoxing, China
Contact / 联系方式: Rojer_Zhang@smecs.com
Usage Scenario / 使用场景: We use Flink CDC to sync Oracle business data to TiDB to build the real-time query OLAP platform.
